eq-author/src/App/history/HistoryItem.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styled from "styled-components";
+import moment from "moment";
+import PropTypes from "prop-types";
+
+const StyledItem = styled.span`
+  display: inline-block;
+  padding: 0 1em 0 0;
+  border: 1px solid #d6d8da;
+  background-color: #f5f5f5;
+  width: 100%;
+`;
+
+const QuestionnaireTitle = styled.div`
+  padding: 1em;
+  color: #333;
+  float: left;
+`;
+
+const QuestionnaireUserName = styled.div`
+  padding: 1em;
+  color: #333;
+  float: right;
+`;
+
+const formatDate = unformattedDate =>
+  moment(unformattedDate).format("DD/MM/YYYY [at] HH:mm");
+
+const HistoryItem = ({ questionnaireTitle, userName, createdAt }) => {
+  return (
+    <StyledItem>
+      <QuestionnaireTitle>
+        {questionnaireTitle} - <strong>Questionnaire created</strong>
```

Comment:
Might be better practice to put `Questionnaire created` in a const value at the top of the page? Makes it easier to find later on if we need to change it and separates business logic from output text.

Reply:
I agree and this is something we need to do throughout the whole app. Normally I've got all output text strings in one file that can be translated and various languages can be used.

Reply:
> I agree and this is something we need to do throughout the whole app. Normally I've got all output text strings in one file that can be translated and various languages can be used.

Love that idea, having them all in the same file. Would make it so much easier to control and change. We'll need to remember to mention it at the retro as something to introduce.

Reply:
> > I agree and this is something we need to do throughout the whole app. Normally I've got all output text strings in one file that can be translated and various languages can be used.
> 
> Love that idea, having them all in the same file. Would make it so much easier to control and change. We'll need to remember to mention it at the retro as something to introduce.

Made a card for it on the retro board: https://trello.com/c/z0Q39jCv